### PR TITLE
revisiting storage blobs and tables naming validation 

### PR DESCRIPTION
**Pull request by shobak101**

### CODE REVIEW

File: src/Microsoft.Azure.WebJobs.Host/Blobs/BlobClient.cs [hunk 1]
```diff
@@ -11,7 +11,7 @@ namespace Microsoft.Azure.WebJobs.Host.Blobs
     internal static class BlobClient
     {
         // Tested against storage service on Jan 2014. All other unsafe and reserved characters work fine.
-        private static readonly char[] UnsafeBlobNameCharacters = { '\\', '[', ']' };
```
Comment (mathewc) [inline src/Microsoft.Azure.WebJobs.Host/Blobs/BlobClient.cs]: Please update the outdated comment.

File: src/Microsoft.Azure.WebJobs.Host/Tables/TableClient.cs [hunk 2]
```diff
@@ -110,7 +110,26 @@ public static void ValidateAzureTableName(string tableName)
 
         public static bool IsValidAzureTableName(string tableName)
         {
-            return Regex.IsMatch(tableName, "^[A-Za-z][A-Za-z0-9]{2,62}$");
+            // Table Name Rules are summerized in this page https://msdn.microsoft.com/en-us/library/azure/dd179338.aspx
+            // Table names may contain only alphanumeric characters.
+            if (!Regex.IsMatch(tableName, "^[A-Za-z][A-Za-z0-9]{2,62}$"))
```
Comment (mathewc) [inline src/Microsoft.Azure.WebJobs.Host/Tables/TableClient.cs]: The above regex seems to already capture the rules you added below.

**Comment by mathewc** [comment]
Fixes: https://github.com/Azure/azure-webjobs-sdk-script/issues/410

File: src/Microsoft.Azure.WebJobs.Host/Tables/TableClient.cs (hunk 2 above)
Comment (shobak101) [inline src/Microsoft.Azure.WebJobs.Host/Tables/TableClient.cs]: actually you are right I misinterpreted it, reverting that change and will only keep the comment referencing the naming rules

Comment (mathewc) [inline src/Microsoft.Azure.WebJobs.Host/Tables/TableClient.cs]: you can remove the if block, but keep your new comment

**Comment by mathewc** [comment]
:ship:

**Pull request closed without merging**